src/app/containers/AssignmentDutyCreateForm.tsx

```diff
@@ -18,11 +17,14 @@ import {
 } from '../api';
 
 // wrapping generic assignment form in redux-form
-const formConfig: ConfigProps<{}, AssignmentDutyFormProps> = {
+const formConfig: ConfigProps<any, AssignmentDutyFormProps> = {
```

Comment:
If we'd like to be typesafe here we can set the type in the ConfigProps to represent the shape of the form data